codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -124,6 +125,11 @@ private String pathToFolder(File file) {
      */
     public void execute() throws MojoExecutionException {
 
+        // iff basePackage is null, empty, or just whitespace reset it to the default value
+        basePackage = StringUtils.defaultString(basePackage, "com.paypal.selion.testcomponents");
```

Comment:
@fakrudeen78 I see your point but I don't agree with the exception approach. I'd rather resort to convention. So, at a minimum a user will need to define a `basePackage` at least one level deep (such as `com`) or accept the default.